beacon-node-oapi.yaml

```diff
@@ -50,8 +58,6 @@ paths:
   /v1/node/health:
     $ref: "./apis/node/health.yaml"
 
-  /validator/{pubkey}:
-    $ref: "./apis/validator/validators.yaml"
   /validator/duties/{epoch}/attester:
     $ref: "./apis/validator/duties.yaml#/attester"
   /validator/duties/{epoch}/proposer:
```

Comment:
The return values here don't match the spreadsheet.  That said, this could just be an array of 32 public keys rather than the map of slot->pubkey as in the current repository if we want to go minimal.

Reply:
I haven't updated /validator endpoints. It will be separate pr.

 That said array won't work if you have small set of validators (common in dev/testnet chains) there can be slots without proposer duties

Reply:
Ah okay; saw it in the PR so figured it was being updated.  Please ignore this and above :)

I thought that we'd always have a proposer for a slot; maybe not, then.

Reply:
I might be wrong, we had some wierd things with just 8 validators but it might have been some bug.

Reply:
You always have a proposer assignment. Proposer's are sampled from the whole active set at each slot _with_ replacement, and proposer slashing is wrt slot rather than epoch

If just one validator, they will be assigned to propose at all slots